2011/04/21 19:02:09 [error] [php] mysql_real_escape_string() [<a href='function.mysql-real-escape-string'>function.mysql-real-escape-string</a>]: Access denied for user 'ODBC'@'localhost' (using password: NO) (D:\projects\mdata\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(27): CMultyData->readRecord()
REQUEST_URI=/mdata/
in D:\projects\mdata\CMultyData.php (156)
in D:\projects\mdata\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (27)
2011/04/21 19:03:18 [error] [php] mysql_real_escape_string() [<a href='function.mysql-real-escape-string'>function.mysql-real-escape-string</a>]: Access denied for user 'ODBC'@'localhost' (using password: NO) (D:\projects\mdata\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(27): CMultyData->readRecord()
REQUEST_URI=/mdata/
in D:\projects\mdata\CMultyData.php (156)
in D:\projects\mdata\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (27)
2011/04/21 19:06:49 [error] [php] mysql_real_escape_string() [<a href='function.mysql-real-escape-string'>function.mysql-real-escape-string</a>]: Access denied for user 'ODBC'@'localhost' (using password: NO) (D:\projects\mdata\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(27): CMultyData->readRecord()
REQUEST_URI=/mdata/
in D:\projects\mdata\CMultyData.php (156)
in D:\projects\mdata\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (27)
2011/04/21 19:06:52 [error] [php] mysql_real_escape_string() [<a href='function.mysql-real-escape-string'>function.mysql-real-escape-string</a>]: Access denied for user 'ODBC'@'localhost' (using password: NO) (D:\projects\mdata\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(27): CMultyData->readRecord()
REQUEST_URI=/mdata/
in D:\projects\mdata\CMultyData.php (156)
in D:\projects\mdata\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (27)
2011/04/21 19:06:53 [error] [php] mysql_real_escape_string() [<a href='function.mysql-real-escape-string'>function.mysql-real-escape-string</a>]: Access denied for user 'ODBC'@'localhost' (using password: NO) (D:\projects\mdata\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(27): CMultyData->readRecord()
REQUEST_URI=/mdata/
in D:\projects\mdata\CMultyData.php (156)
in D:\projects\mdata\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (27)
2011/04/21 19:07:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "site".' in D:\WebServers\home\localhost\www\yii\framework\web\CWebApplication.php:332
Stack trace:
#0 D:\WebServers\home\localhost\www\yii\framework\web\CWebApplication.php(121): CWebApplication->runController('')
#1 D:\WebServers\home\localhost\www\yii\framework\base\CApplication.php(155): CWebApplication->processRequest()
#2 Z:\home\localhost\www\mdata\index.php(24): CApplication->run()
#3 {main} REQUEST_URI=/mdata/
2011/04/21 19:07:42 [error] [php] mysql_real_escape_string() [<a href='function.mysql-real-escape-string'>function.mysql-real-escape-string</a>]: Access denied for user 'ODBC'@'localhost' (using password: NO) (D:\projects\mdata\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(27): CMultyData->readRecord()
REQUEST_URI=/mdata/
in D:\projects\mdata\CMultyData.php (156)
in D:\projects\mdata\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (27)
2011/04/21 19:09:03 [error] [php] mysql_real_escape_string() [<a href='function.mysql-real-escape-string'>function.mysql-real-escape-string</a>]: Access denied for user 'ODBC'@'localhost' (using password: NO) (D:\projects\mdata\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(27): CMultyData->readRecord()
REQUEST_URI=/mdata/
in D:\projects\mdata\CMultyData.php (156)
in D:\projects\mdata\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (27)
2011/04/21 19:09:04 [error] [php] mysql_real_escape_string() [<a href='function.mysql-real-escape-string'>function.mysql-real-escape-string</a>]: Access denied for user 'ODBC'@'localhost' (using password: NO) (D:\projects\mdata\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(27): CMultyData->readRecord()
REQUEST_URI=/mdata/
in D:\projects\mdata\CMultyData.php (156)
in D:\projects\mdata\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (27)
2011/04/21 19:09:05 [error] [php] mysql_real_escape_string() [<a href='function.mysql-real-escape-string'>function.mysql-real-escape-string</a>]: Access denied for user 'ODBC'@'localhost' (using password: NO) (D:\projects\mdata\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(27): CMultyData->readRecord()
REQUEST_URI=/mdata/
in D:\projects\mdata\CMultyData.php (156)
in D:\projects\mdata\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (27)
2011/04/21 19:20:03 [error] [exception.CException] exception 'CException' with message 'The command path "Z:\home\localhost\www\mdata\commands" is not a valid directory.' in Z:\home\localhost\www\yii\framework\console\CConsoleApplication.php:160
Stack trace:
#0 Z:\home\localhost\www\yii\framework\console\CConsoleApplication.php(149): CConsoleApplication->setCommandPath('Z:\home\localho...')
#1 Z:\home\localhost\www\yii\framework\console\CConsoleApplication.php(79): CConsoleApplication->getCommandPath()
#2 Z:\home\localhost\www\yii\framework\base\CApplication.php(141): CConsoleApplication->init()
#3 Z:\home\localhost\www\yii\framework\YiiBase.php(118): CApplication->__construct('config.php')
#4 Z:\home\localhost\www\yii\framework\YiiBase.php(105): YiiBase::createApplication('CConsoleApplica...', 'config.php')
#5 Z:\home\localhost\www\mdata\tests\index.php(24): YiiBase::createConsoleApplication('config.php')
#6 {main}
2011/04/21 19:20:56 [error] [php] mysql_real_escape_string(): Access denied for user 'ODBC'@'localhost' (using password: NO) (Z:\home\localhost\www\mdata\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\tests\index.php(27): CMultyData->readRecord()

in Z:\home\localhost\www\mdata\CMultyData.php (156)
in Z:\home\localhost\www\mdata\CMultyData.php (73)
in Z:\home\localhost\www\mdata\tests\index.php (27)
2011/04/21 19:36:45 [error] [php] include(CMultyData.php): failed to open stream: No such file or directory (Z:\home\localhost\www\yii\framework\YiiBase.php:396)
Stack trace:
#0 Z:\home\localhost\www\mdata\tests\index.php(27): spl_autoload_call()

in Z:\home\localhost\www\mdata\tests\index.php (27)
2011/04/21 19:44:35 [error] [php] include(CMultyData.php): failed to open stream: No such file or directory (Z:\home\localhost\www\yii\framework\YiiBase.php:396)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(27): spl_autoload_call()

in Z:\home\localhost\www\mdata\index.php (27)
2011/04/21 19:46:20 [error] [php] mysql_real_escape_string(): Access denied for user 'ODBC'@'localhost' (using password: NO) (Z:\home\localhost\www\mdata\models\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(28): CMultyData->readRecord()

in Z:\home\localhost\www\mdata\models\CMultyData.php (156)
in Z:\home\localhost\www\mdata\models\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (28)
2011/04/21 19:46:56 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in Z:\home\localhost\www\yii\framework\db\CDbConnection.php:348
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(297): CDbConnection->open()
#1 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(275): CDbConnection->setActive(true)
#2 Z:\home\localhost\www\yii\framework\base\CModule.php(372): CDbConnection->init()
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(426): CModule->getComponent('db')
#4 Z:\home\localhost\www\mdata\models\CMultyData.php(92): CApplication->getDb()
#5 Z:\home\localhost\www\mdata\models\CMultyData.php(42): CMultyData->getDbConnection()
#6 Z:\home\localhost\www\mdata\index.php(27): CMultyData->__construct()
#7 {main}
2011/04/21 19:47:42 [error] [php] mysql_real_escape_string(): Access denied for user 'ODBC'@'localhost' (using password: NO) (Z:\home\localhost\www\mdata\models\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\index.php(28): CMultyData->readRecord()

in Z:\home\localhost\www\mdata\models\CMultyData.php (156)
in Z:\home\localhost\www\mdata\models\CMultyData.php (73)
in Z:\home\localhost\www\mdata\index.php (28)
2011/04/21 19:52:56 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in Z:\home\localhost\www\yii\framework\db\CDbConnection.php:348
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(297): CDbConnection->open()
#1 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(275): CDbConnection->setActive(true)
#2 Z:\home\localhost\www\yii\framework\base\CModule.php(372): CDbConnection->init()
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(426): CModule->getComponent('db')
#4 Z:\home\localhost\www\mdata\models\CMultyData.php(92): CApplication->getDb()
#5 Z:\home\localhost\www\mdata\models\CMultyData.php(42): CMultyData->getDbConnection()
#6 Z:\home\localhost\www\mdata\index.php(27): CMultyData->__construct()
#7 {main}
2011/04/21 19:53:41 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in Z:\home\localhost\www\yii\framework\db\CDbConnection.php:348
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(297): CDbConnection->open()
#1 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(275): CDbConnection->setActive(true)
#2 Z:\home\localhost\www\yii\framework\base\CModule.php(372): CDbConnection->init()
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(426): CModule->getComponent('db')
#4 Z:\home\localhost\www\mdata\models\CMultyData.php(92): CApplication->getDb()
#5 Z:\home\localhost\www\mdata\models\CMultyData.php(42): CMultyData->getDbConnection()
#6 Z:\home\localhost\www\mdata\index.php(27): CMultyData->__construct()
#7 {main}
2011/04/21 19:53:59 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in Z:\home\localhost\www\yii\framework\db\CDbConnection.php:348
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(297): CDbConnection->open()
#1 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(275): CDbConnection->setActive(true)
#2 Z:\home\localhost\www\yii\framework\base\CModule.php(372): CDbConnection->init()
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(426): CModule->getComponent('db')
#4 Z:\home\localhost\www\mdata\models\CMultyData.php(92): CApplication->getDb()
#5 Z:\home\localhost\www\mdata\models\CMultyData.php(42): CMultyData->getDbConnection()
#6 Z:\home\localhost\www\mdata\index.php(27): CMultyData->__construct()
#7 {main}
2011/04/21 19:54:35 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in Z:\home\localhost\www\yii\framework\db\CDbConnection.php:348
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(297): CDbConnection->open()
#1 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(275): CDbConnection->setActive(true)
#2 Z:\home\localhost\www\yii\framework\base\CModule.php(372): CDbConnection->init()
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(426): CModule->getComponent('db')
#4 Z:\home\localhost\www\mdata\models\CMultyData.php(92): CApplication->getDb()
#5 Z:\home\localhost\www\mdata\models\CMultyData.php(42): CMultyData->getDbConnection()
#6 Z:\home\localhost\www\mdata\index.php(27): CMultyData->__construct()
#7 {main}
2011/04/21 19:55:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in Z:\home\localhost\www\yii\framework\db\CDbConnection.php:348
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(297): CDbConnection->open()
#1 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(275): CDbConnection->setActive(true)
#2 Z:\home\localhost\www\yii\framework\base\CModule.php(372): CDbConnection->init()
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(426): CModule->getComponent('db')
#4 Z:\home\localhost\www\mdata\models\CMultyData.php(92): CApplication->getDb()
#5 Z:\home\localhost\www\mdata\models\CMultyData.php(42): CMultyData->getDbConnection()
#6 Z:\home\localhost\www\mdata\index.php(27): CMultyData->__construct()
#7 {main}
2011/04/21 19:57:00 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in Z:\home\localhost\www\yii\framework\db\CDbConnection.php:348
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(297): CDbConnection->open()
#1 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(275): CDbConnection->setActive(true)
#2 Z:\home\localhost\www\yii\framework\base\CModule.php(372): CDbConnection->init()
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(426): CModule->getComponent('db')
#4 Z:\home\localhost\www\mdata\models\CMultyData.php(92): CApplication->getDb()
#5 Z:\home\localhost\www\mdata\models\CMultyData.php(42): CMultyData->getDbConnection()
#6 Z:\home\localhost\www\mdata\index.php(27): CMultyData->__construct()
#7 {main}
2011/04/21 19:57:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in Z:\home\localhost\www\yii\framework\db\CDbConnection.php:348
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(297): CDbConnection->open()
#1 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(275): CDbConnection->setActive(true)
#2 Z:\home\localhost\www\yii\framework\base\CModule.php(372): CDbConnection->init()
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(426): CModule->getComponent('db')
#4 Z:\home\localhost\www\mdata\models\CMultyData.php(92): CApplication->getDb()
#5 Z:\home\localhost\www\mdata\models\CMultyData.php(42): CMultyData->getDbConnection()
#6 Z:\home\localhost\www\mdata\index.php(27): CMultyData->__construct()
#7 {main}
2011/04/21 20:09:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in Z:\home\localhost\www\yii\framework\db\CDbConnection.php:348
Stack trace:
#0 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(297): CDbConnection->open()
#1 Z:\home\localhost\www\yii\framework\db\CDbConnection.php(275): CDbConnection->setActive(true)
#2 Z:\home\localhost\www\yii\framework\base\CModule.php(372): CDbConnection->init()
#3 Z:\home\localhost\www\yii\framework\base\CApplication.php(426): CModule->getComponent('db')
#4 Z:\home\localhost\www\mdata\models\CMultyData.php(92): CApplication->getDb()
#5 Z:\home\localhost\www\mdata\models\CMultyData.php(42): CMultyData->getDbConnection()
#6 Z:\home\localhost\www\mdata\commands\TestCommand.php(15): CMultyData->__construct()
#7 [internal function]: TestCommand->actionInit()
#8 Z:\home\localhost\www\yii\framework\console\CConsoleCommand.php(135): ReflectionMethod->invokeArgs(Object(TestCommand), Array)
#9 Z:\home\localhost\www\yii\framework\console\CConsoleCommandRunner.php(63): CConsoleCommand->run(Array)
#10 Z:\home\localhost\www\yii\framework\console\CConsoleApplication.php(88): CConsoleCommandRunner->run(Array)
#11 Z:\home\localhost\www\yii\framework\base\CApplication.php(155): CConsoleApplication->processRequest()
#12 Z:\home\localhost\www\mdata\index.php(25): CApplication->run()
#13 {main}
2011/04/21 20:09:57 [error] [php] mysql_real_escape_string(): Access denied for user 'ODBC'@'localhost' (using password: NO) (Z:\home\localhost\www\mdata\models\CMultyData.php:156)
Stack trace:
#0 Z:\home\localhost\www\mdata\commands\TestCommand.php(16): CMultyData->readRecord()
#1 unknown(0): TestCommand->actionInit()
#2 Z:\home\localhost\www\yii\framework\console\CConsoleCommand.php(135): ReflectionMethod->invokeArgs()
#3 Z:\home\localhost\www\yii\framework\console\CConsoleCommandRunner.php(63): TestCommand->run()
#4 Z:\home\localhost\www\yii\framework\console\CConsoleApplication.php(88): CConsoleCommandRunner->run()
#5 Z:\home\localhost\www\yii\framework\base\CApplication.php(155): CConsoleApplication->processRequest()
#6 Z:\home\localhost\www\mdata\index.php(25): CConsoleApplication->run()

in Z:\home\localhost\www\mdata\models\CMultyData.php (156)
in Z:\home\localhost\www\mdata\models\CMultyData.php (73)
in Z:\home\localhost\www\mdata\commands\TestCommand.php (16)
